MapboxMobileEvents/MMEConstants.m

```diff
@@ -71,6 +71,8 @@
 
 NSString * const MMEEventSource = @"mapbox";
 
+NSString * const MMELoggerHTML = @"<html><head><script type='text/javascript' src='https://www.gstatic.com/charts/loader.js'></script><script type='text/javascript'>google.charts.load('current', {'packages':['timeline']});var dataString;function addData(data) {dataString = data}window.webkit.messageHandlers.data.postMessage('data');google.charts.setOnLoadCallback(drawChart); function drawChart() {var dataTable = new google.visualization.DataTable({cols: [{id: 'eventType', label: 'Event Type', type: 'string'},{id: 'start', label: 'Event Start Time', type: 'datetime'},{id: 'end', label: 'Event End Time', type: 'datetime'}],rows: dataString});var options = {'title':'Telemetry Log Data','width':1024,'height':400,'timeline': { groupByRowLabel: true }};var chart = new google.visualization.Timeline(document.getElementById('chart_div'));google.visualization.events.addListener(chart, 'ready', afterDraw);chart.draw(dataTable, options);}function afterDraw() {window.webkit.messageHandlers.complete.postMessage('complete');}</script></head><body><div id='chart_div'></div></body></html>";
```

Comment:
Thanks for putting this back and sorry for waffling from [my original comment](https://github.com/mapbox/mapbox-events-ios/pull/26/files/9956e4b2cb1e5fab798568b2ed2280c87e32bba9#r154723787). No need to change anything here but, in the future to make this more readable and easier to update, we could code generate this string from an html file that is in this project but not shipped as part of this library.